client/store-management-links/quick-link-category/test/index.js

```diff
@@ -0,0 +1,22 @@
+/**
+ * External dependencies
+ */
+import { render } from '@testing-library/react';
+import '@testing-library/jest-dom';
+
+/**
+ * Internal dependencies
+ */
+import { QuickLinkCategory } from '..';
+
+describe( 'QuickLinkCategory', () => {
+	it( 'displays the passed title and children', () => {
```

Comment:
I was expecting to see an assertion for the title being displayed. Did that get missed?

Reply:
It sure did! Thanks for spotting that.